logs-lib/utils.libsonnet

```diff
@@ -10,6 +10,9 @@
 
   labelsToPanelLegend(labels): std.join('/', ['{{%s}}' % [label] for label in labels]),
 
+  toSentenceCase(string)::
```

Comment:
Nice useful util!

Reply:
I'll try to move those utils to grafonnet some time